website/blog/2020-07-06-version-0.63.md

```diff
@@ -0,0 +1,153 @@
+- **Syntax Error Formatting**: We’ve improved the formatting for syntax errors and added codeframes with syntax highlighting so you can see the source of the error, fix it, and continue coding without React Native getting in your way.
+
+We’ve wrapped all of these features into an improved visual design that’s consistent between errors and warnings and allows paginating through all logs in one enjoyable UI.
+
+With this change we’re also deprecating YellowBox in favor of LogBox APIs:
+
+- `LogBox.ignoreLogs()`: This function replaces `YellowBox.ignoreLogs([])` as a way to silence any logs that match the given strings or regexes.
+- `LogBox.ignoreAllLogs()`: This function replaces `console.disableYellowBox` as a way to turn off error or warning notifications. Note: this only disables notifications, uncaught errors will still open a full screen LogBox.
+
+In 0.63, we will warn when using these deprecated modules or methods. Please update your call sites off of these APIs before they are removed in 0.64.
+
+For more information on LogBox and debugging react native, see the docs [here](https://reactnative.dev/docs/debugging#in-app-errors-and-warnings).
+
+## Pressable
+
+React Native is built to enable your applications to meet user’s expectations of the platform. This includes avoiding “tells”—little things that give away that the experience was built with React Native. One major source of these tells has been the Touchable components: `Button`, `TouchableWithoutFeedback`, `TouchableHighlight`, `TouchableOpacity`, `TouchableNativeFeedback`, and `TouchableBounce`.
+
+These components make it easy to get started making things interactive. However, because they include built-in styles and effects that don’t match the system interaction, they make it easy to tell when experiences are written with React Native. Further, as React Native has grown and our bar for high-quality applications has gone up, these components haven't grown with it. React Native now supports platforms like Web, Desktop, and TV, but support for additional input modalities has been lacking. React Native needs something that meets users' expectations and gives them the best possible interaction experience on all platforms.
+
+To address these problems, we are shipping a new core component called Pressable. This component can be used to detect various types of interactions. It can be extended by the platform to include things like hover, blur, focus, and more. We expect that most people will build and share components utilizing Pressable under the hood instead of relying on the default experience of something like TouchableOpacity.
+
+```jsx
+import { Pressable, Text } from 'react-native';
+
+<Pressable
+  onPress={() => {
+    console.log('pressed');
+  }}
+  style={({ pressed }) => ({
+    backgroundColor: pressed ? 'lightskyblue' : 'white'
+  })}>
+  <Text style={styles.text}>Press Me!</Text>
+</Pressable>;
+```
+
+<p style="text-align: center; font-size: 12px">
```

Comment:
It would be nice to extract inline style to CSS file and use `class` attribute since this could be useful for future posts (and generally for adding image captions in other places). There are also few occurrences (repetitions) of this paragraph styling in this post.

Reply:
Yeah, I can do that! I was surprised there was a missing class for this!